src/views/product-tutorials-view/components/sitemap/index.tsx

```diff
@@ -9,7 +9,7 @@ import s from './sitemap.module.css'
 export function ProductTutorialsSitemap({
   collections,
 }: {
-  collections: ProductPageData['pageData']['allCollections']
+  collections: ProductPageData['allCollections']
```

Comment:
Have moved `allCollections` up a level - I think it's helpful if we keep `pageData` as authored, and enhance it (in the front-end for now) with stuff like `inlineCollections`, `inlineTutorials`, and `allCollections`.